### ISSUE TITLE
Textbox freezes

### ISSUE THREAD

**Issue by GoogleCodeExporter** [KGIII/pino-twitter#45]
```
What steps will reproduce the problem?
1. Was writing a reply.
2. Was writing a normal tweet.

I don't have more info about it.

What is the expected output? What do you see instead?
When i type enter to send the tweet says "connection error". It would be 
fine if it recovered what was in the text box or something, but instead of 
that the input box gets disabled and i can't rewrite the tweet or write 
another. When i do Ctrl + N the text box is cleaned but keeps disabled so i 
have to close and re-open Pino.

What version of the product are you using? On what operating system?
o.1.3 on ubuntu karmic.
```

Original issue reported on code.google.com by `mich.ba...@gmail.com` on 13 Jan 2010 at 10:33

**Comment by GoogleCodeExporter** [comment]
```
yeah! i had same problem & the fix is too simple just needs some changes in
main_window.vala:747
```

Original comment by `everpl...@gmail.com` on 14 Jan 2010 at 9:43

Attachments:
- [pino.diff](https://storage.googleapis.com/google-code-attachments/pino-twitter/issue-45/comment-1/pino.diff)

**Comment by GoogleCodeExporter** [comment]
Original comment by `tro...@gmail.com` on 14 Jan 2010 at 10:57
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
Closed by cb9e2c3a53 revision.
```

Original comment by `tro...@gmail.com` on 27 Jan 2010 at 4:21
- Changed state: **Fixed**
